dace/runtime/include/dace/perf/reporting.h

```diff
@@ -112,7 +124,19 @@ namespace perf {
             int state_id,
             int el_id
         ) {
-            std::thread::id tid = std::this_thread::get_id();
+            add_completion(name, cat, tstart, tend, -1, sdfg_id, state_id, el_id);
```

Comment:
same as the above comment

Reply:
Same as above, the same solution applies imo.